safe/test/test_translations.py

```diff
@@ -136,6 +136,43 @@ def test_qgis_translations(self):
         message = 'expected %s but got %s' % (expected_message, real_message)
         self.assertEqual(expected_message, real_message, message)
 
+    @unittest.skipIf(
```

Comment:
In this file, can you also clean these 'Xtest'. We put an X to avoid travis to run them. We can't add the decorator for some reason but it seems your new test is covering these tests already, so maybe you can remove them? It would be nice. Thanks